java/test/jmri/jmrit/operations/rollingstock/engines/tools/PrintEngineRosterFrameTest.java

```diff
@@ -25,9 +25,12 @@ public void testPrintPreview() {
         JUnitOperationsUtil.initOperationsData();
         EnginesTableFrame ctf = new EnginesTableFrame();
 
+        for (int i = 0; i <= ctf.enginesModel.SORTBY_COMMENT; i++) {
+
         PrintEngineRosterFrame f = new PrintEngineRosterFrame(true, ctf);
         Assert.assertNotNull("exists", f);
```

Comment:
No need to change, but the indentation is wrong after you added the `for` loop above. This applies down to line 44.